python/ccp.py

```diff
@@ -3,6 +3,30 @@
 import struct
 from enum import IntEnum, Enum
 
+from typing import TypedDict, Optional
+
+class ExchangeStationIdsReturn(TypedDict):
+    id_length: int
+    data_type: int
+    available: int  
+    protected: int  
+
+class GetDaqListSizeReturn(TypedDict):
+    list_size: int
+    first_pid: int
+
+class GetSessionStatusReturn(TypedDict):
+    status: int
+    info: Optional[int]   
+
+class DiagnosticServiceReturn(TypedDict):
+    length: int
+    type: int
+
+class ActionServiceReturn(TypedDict):
+    length: int
+    type: int
```

Comment:
if we're gonna do this, might as well just return the class instead of a dict. what do you think @gregjhogan?